app/controllers/provider/admin/registry/policies_controller.rb

```diff
@@ -1,7 +1,7 @@
 #frozen_string_literal: true
 
 class Provider::Admin::Registry::PoliciesController < Provider::Admin::BaseController
-  activate_menu :account, :integrate, :custom_policies
+  activate_menu :account, :integrate
```

Comment:
Don't we remove the whole controller then?

Reply:
Good point, and you're probably right. That controller isn't used anywhere else, is it?

Reply:
Remove it, it's not used. We are using `PoliciesService` in`integrations_controller` to fetch APIcast registry.

Reply:
OK. How about [app/controllers/admin/api/registry/policies_controller.rb](https://github.com/3scale/porta/blob/master/app/controllers/admin/api/registry/policies_controller.rb)?

Reply:
If the API is not documented / published... remove it as well.

Reply:
I think app/controllers/admin/api/registry/policies_controller.rb _is_ documented and published to all Saas accounts. Even if no one is using it, removing it can be trickier. I'd recommend checking with @3scale/product, as they may want to issue a note about it.